test/unit/lodash.spec.js

```diff
@@ -0,0 +1,522 @@
+
+        describe('ceil', function () {
+
+            var params, area;
+
+            beforeEach(function () {
+                params = datasets.ceil();
+                area = params[0];
+            });
+
+            it('should verify params', function () {
+                expect(area instanceof Function).to.eql(true);
+                expect(params).to.have.length(2);
+            });
+
+            it('should return area of rectangle when area side is smaller than 0 return 0', function () {
+                expect(area(3.2,4.6)).to.eql(15.6);
+                expect(area(-2, 4)).to.eql(0);
+                expect(area(0, 0)).to.eql(0);
+                expect(area(-2, 4)).to.eql(0);
+            });
+
+            it('should return round up area of rectangle', function () {
+                expect(_.ceil.apply(_,([area(3.2, 4.4)]))).to.eql(16);
+                expect(_.ceil.apply(_,([area(2.33, 0.55)]))).to.eql(6);
+                expect(_.ceil.apply(_,([area(1.74, 0.54), 4]))).to.eql(4.5601);
+                expect(_.ceil.apply(_,([area(-2, 0.54), 4]))).to.eql(0);
+            });
+        });
+
+        describe('divide', function () {
+
+            var params;
+
+            beforeEach(function () {
+                params = datasets.divide();
+            });
+
+            it('should verify params', function () {
+                _(params[0]).forEach(function (value) {
+                    expect(typeof value).to.eql('number');
+                });
+
+                expect(params).to.have.length(2);
+            });
+
+            it('should divide two numbers a and b', function () {
+                expect(_.divide.apply(_, params)).to.eql(Infinity);
+            });
+
+        });
+
+        describe('floor', function () {
+
+            var params;
+            var capacity;
+
+            beforeEach(function () {
+                params = datasets.floor();
+                capacity = params[0];
+            });
+
+            it('should verify params', function () {
+                expect(capacity instanceof Function).to.eql(true);
+            });
+
+            it('should return capacity of cuboid', function () {
+                expect(capacity(2, 2, 2)).to.eql(8);
+                expect(capacity(2.3, 2.25, 5.43)).to.eql(28.10025);
+                expect(capacity(-1, 2.25, 5.43)).to.eql(0);
+                expect(capacity(3, -2, 5.43)).to.eql(0);
+                expect(capacity(3, 2, -3)).to.eql(0);
+            });
+
+            it('should return number round down', function () {
+                expect(_.floor.apply(_, [capacity(2,2,2)])).to.eql(8);
+                expect(_.floor.apply(_, [capacity(2.2,3.3,4.4), 2])).to.eql(31.94);
+            });
+        });
+
+        describe('max', function () {
+
+            var params;
+
+            beforeEach(function () {
+                params = datasets.max();
+            });
+
+            it('should find max value (5) in user array', function () {
```

Comment:
Do not provide user with such obvious test case name, they should be able to tell from the test that 5 is the required number.